twingate/provider.go

```diff
@@ -63,56 +65,57 @@ func Provider(version string) *schema.Provider {
 func providerOptions() map[string]*schema.Schema {
 	return map[string]*schema.Schema{
 		attr.APIToken: {
-			Type:        schema.TypeString,
-			Optional:    true,
-			Sensitive:   true,
-			DefaultFunc: schema.EnvDefaultFunc(EnvAPIToken, nil),
```

Comment:
Note to self - we remove default from schema because new plugins sdk doesnt support it.
So these default values moved to `configure` func